build-deploy-develop/toolkit/api-libraries/.pages

```diff
@@ -0,0 +1,7 @@
+title: API Libraries
+nav:
+  - index.md
+  - 'Polkadot.js SDK (JavaScript/TypeScript)': polkadot-js-sdk.md
+  - 'PAPI (JavaScript/TypeScript)': papi.md
+  - 'Sidecar (Rest API)': sidecar.md
+  - 'PY-Substrate-Interface (Python)': py-substrate-interface.md
```

Comment:
```suggestion
  - 'Polkadot.js SDK (JavaScript/TypeScript)': polkadot-js-sdk.md
  - 'PAPI (JavaScript/TypeScript)': papi.md
  - 'Sidecar (Rest API)': sidecar.md
  - 'Python Substrate Interface (Python)': py-substrate-interface.md
```

It's kinda weird it says python twice. I'd honestly prefer if we didn't use the parenthesis in the titles. But I'm just gonna go with the flow here. Can always change later if need be 🤷‍♀️